hieradata/class/ci_master.yaml

```diff
@@ -190,3 +190,7 @@ govuk_jenkins::plugins:
       version: "2.11"
     ws-cleanup:
       version: "0.32"
+    next-build-number:
+      version: "1.4"
+    job-dsl:
+      version: "1.53"
```

Comment:
Do we need the `job-dsl` plugin? It's listed as an optional dependency and is only needed to change the build number from within a job DSL script which I don't think we do anywhere.